api/src/endpoints/svgThumbnail.js

```diff
@@ -0,0 +1,44 @@
+import express from 'express';
+import fs from 'fs';
```

Comment:
unused module?

Reply:
My bad, forgot to remove it after refactoring.